tools/serverpod_cli/lib/src/generator/generator.dart

```diff
@@ -1,104 +1,87 @@
-import 'package:code_builder/code_builder.dart';
-import 'package:dart_style/dart_style.dart';
-
-import 'class_analyzer.dart';
-import 'class_generator.dart';
-import 'code_cleaner.dart';
-import 'config.dart';
-import 'protocol_analyzer.dart';
-import 'protocol_generator.dart';
-import 'code_analysis_collector.dart';
+import 'package:serverpod_cli/analyzer.dart';
+import 'package:serverpod_cli/src/generator/code_generation_collector.dart';
+import 'package:serverpod_cli/src/generator/code_generator.dart';
+import 'package:serverpod_cli/src/util/print.dart';
 
+/// Analyze the server package and generate the code.
 Future<void> performGenerate({
   required bool verbose,
   bool dartFormat = true,
-  bool requestNewAnalyzer = true,
   String? changedFile,
+  required GeneratorConfig config,
+  required ProtocolEndpointsAnalyzer endpointsAnalyzer,
 }) async {
-  if (!config.load()) return;
-
-  String generator(spec) => generateCode(spec, dartFormat);
-
-  print('Running serverpod generate.');
-
-  var collector = CodeAnalysisCollector();
+  var collector = CodeGenerationCollector();
 
   if (verbose) {
-    print('Analyzing protocol yaml files.');
+    printww('Analyzing entities in the protocol directory...');
```

Comment:
serializable entities

Reply:
And skip the tripple dots. :)

Reply:
979e55ab

Remove `...` in all other messages, too?